pwndbg/commands/heap.py

```diff
@@ -91,11 +93,37 @@ def arena(addr=None):
     main_heap   = pwndbg.heap.current
     main_arena  = main_heap.get_arena(addr)
 
-    if main_arena == None:
+    if main_arena is None:
         return
 
     print(main_arena)
 
+
+@pwndbg.commands.ParsedCommand
+@pwndbg.commands.OnlyWhenRunning
+def arenas():
+    """
+    Prints out allocated arenas 
+    """
+
+    heap  = pwndbg.heap.current
+    addr  = None
+    arena = heap.get_arena(addr)
+    main_arena_addr = int(arena.address)
+    fmt = '[%%%ds]' % (pwndbg.arch.ptrsize *2)
+    while addr != main_arena_addr:
+        
+        h = heap.get_region(addr)
+        if not h:
+            print(red('Could not find the heap'))
+            return
+        
+        print(fmt%(hex(addr) if addr else 'main'), h)
```

Comment:
Note: I have merged @zachriggle's debugging-works branch locally to be able to use `set exceptions-verbose on`.